pkg/platform/kube/logProxy/elastic/opensearch.go

```diff
@@ -48,23 +53,249 @@ func NewOpenSearchLogProxy(config *platformconfig.ElasticSearchConfig) (*OpenSea
 		// Set to true to skip TLS verification if SSLVerificationMode is "none"
 		InsecureSkipVerify: config.SSLVerificationMode == "none",
 	}
-	if openSearchClient.client, err = opensearch.NewClient(opensearch.Config{
+	if openSearchClient.client, err = opensearchapi.NewClient(opensearchapi.Config{Client: opensearch.Config{
```

Comment:
`opensearchapi` supports both AWS and vanilla OS, but `opensearch` only vanilla?

Reply:
@TomerShor no, they both support both. It's just that opensearchapi provides more typed search entities.